concepts.tex

```diff
@@ -127,6 +127,209 @@
 which requires \tcode{C<T>()}.
 \exitexample
 
+\rSec1[concepts.lib.synopsis]{Header \tcode{<experimental/ranges/concepts>} synopsis}
```

Comment:
With the synopsis, [concepts.lib.general]/3 is now redundant and should be removed.